dist/c89-compatible/libintvector.h

```diff
@@ -1,533 +1,554 @@
-  (vst1q_u32((uint32_t*)(x0),(vrev32q_u8(x1))))
-
-#define Lib_IntVector_Intrinsics_vec128_store64_be(x0, x1)	\
-  (vst1q_u32((uint32_t*)(x0),(vrev64q_u8(x1))))
-
-#define Lib_IntVector_Intrinsics_vec128_insert8(x0, x1, x2)	\
-  (vsetq_lane_u8(x1,x0,x2))
-
-#define Lib_IntVector_Intrinsics_vec128_insert32(x0, x1, x2)	\
-  (vsetq_lane_u32(x1,x0,x2))
-
-#define Lib_IntVector_Intrinsics_vec128_insert64(x0, x1, x2)	\
-  (vreinterpretq_u32_u64(vsetq_lane_u64(x1,vreinterpretq_u64_u32(x0),x2)))
-
-#define Lib_IntVector_Intrinsics_vec128_extract8(x0, x1)	\
-  (vgetq_lane_u8(x0,x1))
-
-#define Lib_IntVector_Intrinsics_vec128_extract32(x0, x1)	\
-  (vgetq_lane_u32(x0,x1))
-
-#define Lib_IntVector_Intrinsics_vec128_extract64(x0, x1)	\
-  (vreinterpretq_u32_u64(vgetq_lane_u64(vreinterpretq_u64_u32(x0),x1)))
-
-#define Lib_IntVector_Intrinsics_vec128_zero  \
-  (vdup_n_u8(0))
-
-#define Lib_IntVector_Intrinsics_vec128_add64(x0, x1) \
-  (vreinterpretq_u32_u64(vaddq_u64(vreinterpretq_u64_u32(x0), vreinterpretq_u64_u32(x1))))
-
-#define Lib_IntVector_Intrinsics_vec128_sub64(x0, x1)		\
-  (vreinterpretq_u32_u64(vsubq_u64(vreinterpretq_u64_u32(x0), vreinterpretq_u64_u32(x1))))
-
-#define Lib_IntVector_Intrinsics_vec128_mul64(x0, x1) \
-  (vmull_u32(vmovn_u64(x0), vmovn_u64(x1)))
-
-#define Lib_IntVector_Intrinsics_vec128_smul64(x0, x1) \
-  (vmull_u32(vmovn_u64(x0), vdupq_n_u64(x1)))
-
-#define Lib_IntVector_Intrinsics_vec128_add32(x0, x1) \
-  (vaddq_u32(x0, x1))
-
-#define Lib_IntVector_Intrinsics_vec128_sub32(x0, x1)		\
-  (vsubq_u32(x0, x1))
-
-#define Lib_IntVector_Intrinsics_vec128_mul32(x0, x1) \
-  (vmulq_lane_u32(x0, x1))
-
-#define Lib_IntVector_Intrinsics_vec128_smul32(x0, x1) \
-  (vmulq_lane_u32(x0, vdupq_n_u32(x1)))
-
-#define Lib_IntVector_Intrinsics_vec128_load128(x) \
-  ((uint32x4_t)(x))
-
-#define Lib_IntVector_Intrinsics_vec128_load64(x) \
-  (vreinterpretq_u32_u64(vdupq_n_u64(x))) /* hi lo */
-
-#define Lib_IntVector_Intrinsics_vec128_load32(x) \
-  (vdupq_n_u32(x)) /* hi lo */
-
-static inline Lib_IntVector_Intrinsics_vec128 Lib_IntVector_Intrinsics_vec128_load64s(uint64_t x1, uint64_t x2){
-  const uint64_t a[2] = {x1,x2};
-  return vreinterpretq_u32_u64(vld1q_u64(a));
-}
-
-static inline Lib_IntVector_Intrinsics_vec128 Lib_IntVector_Intrinsics_vec128_load32s(uint32_t x1, uint32_t x2, uint32_t x3, uint32_t x4){
-  const uint32_t a[4] = {x1,x2,x3,x4};
-  return vld1q_u32(a);
-}
-
-#define Lib_IntVector_Intrinsics_vec128_interleave_low32(x1, x2) \
-  (vzip1q_u32(x1,x2))
-
-#define Lib_IntVector_Intrinsics_vec128_interleave_high32(x1, x2) \
-  (vzip2q_u32(x1,x2))
-
-#define Lib_IntVector_Intrinsics_vec128_interleave_low64(x1,x2) \
-  (vreinterpretq_u32_u64(vzip1q_u64(vreinterpretq_u64_u32(x1),vreinterpretq_u64_u32(x2))))
-
-#define Lib_IntVector_Intrinsics_vec128_interleave_high64(x1,x2) \
-  (vreinterpretq_u32_u64(vzip2q_u64(vreinterpretq_u64_u32(x1),vreinterpretq_u64_u32(x2))))
-
-#endif
-#endif
+#ifndef __Vec_Intrin_H

```

Comment:
I think all lines in this file show as a diff because of a line-ending discrepancy. Can you convert line-endings to the right format (I think dos2unix will do)?

Reply:
I ran dos2unix and checked in.